src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrEnumerateDomainsInSamServerRequest.java

```diff
@@ -0,0 +1,40 @@
+/***************************************************************************
+ * COPYRIGHT (C) 2017, Rapid7 LLC, Boston, MA, USA.
+ * All rights reserved. This material contains unpublished, copyrighted
+ * work including confidential and proprietary information of Rapid7.
+ **************************************************************************/
+package com.rapid7.client.dcerpc.mssamr.messages;
+
+import java.io.IOException;
+import com.rapid7.client.dcerpc.io.PacketOutput;
+import com.rapid7.client.dcerpc.messages.RequestCall;
+import com.rapid7.client.dcerpc.mssamr.objects.ServerHandle;
+
+/**
+ *
+ */
+public class SamrEnumerateDomainsInSamServerRequest extends RequestCall<SamrEnumerateDomainsInSamServerResponse> {
+    public final static short OP_NUM = 6;
+    private final ServerHandle serverHandle;
+    private final int enumContext;
+    private final int maxLength;
+
+    public SamrEnumerateDomainsInSamServerRequest(ServerHandle handle, int enumContext, int maxLength) {
```

Comment:
it it fine start with `{` ?

Reply:
@sangdo-lee What do you mean?

Reply:
oh sorry, never mind it. before I thought {  was started with new line..

Reply:
Yea @tosmun-r7 had a big commit where we standardize our code format. We always keep the { on the same line from now on in smbj-rpc.